### DISCUSSION TITLE
How to view usage?

### DISCUSSION THREAD

**Discussion by ccocks** [community/community#140223] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to view how many requests are done for each model.

**Comment by Akash1134** [comment]
Hi @blazgocompany,

GitHub provides free API usage so that you can experiment with AI models in your own application. The free API usage is rate limited. If you get rate limited, you will need to wait for the rate limit that you hit to reset before you can make more requests. See [Rate limits](https://docs.github.com/en/github-models/prototyping-with-ai-models#rate-limits) to know the requests one can make for each model. 

Hope that helps.
Thanks!

**Comment by ccocks** [comment]
@Akash1134 No, thats pretty self explanatory from the docs. My question was that how to keep track of how many I've used in a day. e.g. The gpt-4o model has a rate limit of 50 a day, but I want to keep track of how many are used.
